docs/contributions/README.md

```diff
@@ -0,0 +1,73 @@
+
+## Fixing a Bug or Adding a Feature
+
+MDC React's goal is to build in parity of [MDC Web](https://github.com/material-components/material-components-web). MDCR's features should have a 1:1 mapping within the constraints of React. If there is part of the API you disagree with please open a [new issue](https://github.com/material-components/material-components-web-react/issues/new). Depending on the content it may fall to MDC Web's plate, which trickles to MDCR a release later.
+
+**When in doubt open an issue. We're here to _hear_ you!**
+
+### Add a Feature
+
+We consider any new API to be a new feature. An API is any of the following:
+
+* Update to React Component APIs
+* Other hooks into React Component
+* Adding a new component
+
+If changes fall under these categories or you'd like to add a new component please open an [issue](https://github.com/material-components/material-components-web-react/issues/new) _first_. The team wants to collaborate with you before you dive head first into making changes and updates.
+
+> If the changes don't fall into the above categories, or fix an existing feature, it most likely will fall into MDC Web
+
+### Fixing Bugs
+
+* If the issue is a small doc change (READMEs, documentation, etc.) please go ahead and open a [pull request](https://github.com/material-components/material-components-web-react/pulls).
+* If the changes you want to perform are under 10 - 20 lines of code, please open a [pull request](https://github.com/material-components/material-components-web-react/pulls).
+* Anything larger or change to an API will require an issue to be opened first.
+
+### Commit Message Format
+
+The final commit message to the mdc-foo package, for GitHub issue 1234, should look like this:
+
+```
+# for a fix to an issue
+fix(foo): Short description of fix
+
+# for a new feature
+feat(foo): Short description of feature
+
+# for a doc update
+docs(foo): Short description of doc changes
+```
+
+This commit message is pulled into our CHANGELOG when we [release](../release.md) and is based on [Angular's Git commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits). If you have any difficulty, please reach out to us on [Discord](https://discord.gg/material-components) or add a comment to the GitHub issue or PR.
+
+## Pull Requests
+
+Pull requests should meet the following criteria:
+
+* **PRs should be focused!** PRs should focus on fixing one issue or an additional feature. Anything extra requires another PR.
+* If there is an existing [issue](https://github.com/material-components/material-components-web-react/issues) please refer to it in the description of your PR.
+* Please also add notes/description about what your changes are trying to achieve (or anything you've learned). Brevity appreciated.
+
+### Process
+
+Before opening a PR, it should be up to date with targeted release (rc<release_number> ie. rc7.0, rc8.0, etc.). In most cases it will never be branched from _master_.
```

Comment:
We should add `0.` in front of the versions in the branch name moving forward, like `rc0.7.0`, `rc0.8.0`.
So it's not confusing if we ever release `1.0.0` lol 😱

Reply:
ya true - just thought it was easier to type lol